kimetaro.py

```diff
@@ -87,9 +99,27 @@
 @pysnooper.snoop()
 def add(message):
```

Comment:
This function is too complicated.

It's better to write to know the following
- The function accept 2 types of string
  - `"str"...`
  - `str(space)str`
- The (if) condition is for which the type of string

Reply:
Fixed up.